CONTRIBUTING.md

```diff
@@ -34,26 +36,39 @@ on GitHub if you don't have a token.
    helm repo update
    ```
 
-5. Install JupyterHub in minikube with helm
-
-        ./testing/minikube/install-hub
-
-6. Install binderhub and its development requirements:
+5. Install binderhub and its development requirements:
 
-        python3 -m pip install -e . -r dev-requirements.txt
+      python3 -m pip install -e . -r dev-requirements.txt
+          
+  This list of packages is necessary to create an environment that will generate the Docker image using the Git repository. Regardless of what is in the setup.py file, it will install what the user needs to build the Docker image.
```

Comment:
The list of packages in `dev-requirements.txt` will be used to create a suitable environment for generating the Docker image for the  Git Repository. Regardless ... ~it~  the requirements file will ...